### PR TITLE
RebillyProcessor sandbox integrated MPI

### PR DESCRIPTION
**Pull request by Zaphell**
Adds ability to select RebillyProcessor MPI for integrated 3DS

### CODE REVIEW

File: scripts/deploy-branch.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var path = require('path');
 set('-e');
 set('-v');
 
-var branch = process.env.TRAVIS_BRANCH && process.env.TRAVIS_BRANCH.toLowerCase();
+var branch = process.env.TRAVIS_BRANCH && process.env.TRAVIS_BRANCH;
```
Comment (adamaltman) [inline scripts/deploy-branch.js]: Why this change?

Comment (Zaphell) [inline scripts/deploy-branch.js]: This made the preview link unavailable to my branch because it had uppercase lettering (GWY-5495)
